.github/workflows/lint.yml

```diff
@@ -14,18 +14,15 @@ permissions:
   contents: read
 
 jobs:
-  bazel_to_cmake:
+  pre-commit:
```

Comment:
I'll flip the "required" checks in the repository settings before merging this:

![image](https://github.com/iree-org/iree/assets/4010439/7b8ff470-48b0-45eb-9367-1598479f5797)

![image](https://github.com/iree-org/iree/assets/4010439/ed511755-2f26-414d-8e52-8cf7f8917ba3)